ci/scripts/comment-on-pr-with-pkl-pipeline-diffs.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/ash
+# shellcheck shell=dash
```

Comment:
Shellcheck continually gives a warning that ash is being treated as dash, and says to add this to suppress the warning. I don't like pointless warnings so I added it :)